client/Dockerfile

```diff
@@ -1,6 +1,5 @@
 FROM node:16-alpine
 ENV PATH /app/node_modules/.bin:$PATH
 WORKDIR /app
-COPY . .
```

Comment:
looks like this got accidentally removed.

Reply:
Resolved in #145